src/yellow/components/button/_index.scss

```diff
@@ -6,7 +6,7 @@
   cursor: pointer;
   font-size: $size-font-xxs;
   font-weight: $font-style-weight-bold;
-  min-width: 280px;
+  min-width: 160px;
```

Comment:
Talking to the designers, we agreed that the minimum button width should be 160px.